_config.yml

```diff
@@ -30,17 +30,35 @@ search_tokenizer_separator: /[\s/]+/
 
 # Aux links for the upper right navigation
 aux_links:
-  "GrimoireLab on GitHub":
+  "Website":
+    - "//chaoss.github.io/grimoirelab"
+  "GitHub":
     - "//github.com/chaoss/grimoirelab"
+  "Mailing List":
+    - "//lists.linuxfoundation.org/mailman/listinfo/grimoirelab-discussions"
+  "Slack":
+    - "//chaoss-workspace.slack.com/archives/C022NPTPC8Z"
 
 # Heading anchor links appear on hover over h1-h6 tags in page content
 # allowing users to deep link to a particular heading on a page.
 #
 # Supports true (default) or false/nil
 heading_anchors: true
 
+# Back to top link
+back_to_top: true
+back_to_top_text: "Back to top"
+
 # Footer content appears at the bottom of every page's main content
-footer_content: 'Copyright &copy CHAOSS; 2017-2021. Distributed by an <a href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.'
+footer_content: 'Copyright &copy CHAOSS; 2017-2021. <br>Distributed by an <a href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.'
```

Comment:
According to the docs, it seems this way of setting footer is deprecated. But they did mention it is still supported. So @vchrombie what do you think about it ?

Reply:
> According to the docs, it seems this way of setting footer is deprecated. But they did mention it is still supported. So @vchrombie what do you think about it ?

I think it still supports, we can use it as long as it supports.

Reply:
Yep. That's fine with me :+1: